android/app/src/main/java/com/reactnativenavigation/activities/BaseReactActivity.java

```diff
@@ -354,4 +354,18 @@ public void switchToTab(ReadableMap params) {
     public void toggleTabs(ReadableMap params) {
         return;
     }
+
+    public void toggleNavigationBar(ReadableMap params) {
+        if (mToolbar == null) {
+            return;
+        }
+
+        boolean hide = params.getBoolean(KEY_HIDDEN);
+        boolean animated = params.getBoolean(KEY_ANIMATED);
+        if (hide) {
+            mToolbar.hideToolbar(animated);
```

Comment:
`hideToolbar` and `showToolbar` don't animate the transition since they only change the View's visibility to `GONE` or `VISIBLE`. I'll fix it after we merge.